Touch Bar Simulator/Glue.swift

```diff
@@ -43,3 +43,24 @@ extension NSMenuItem {
 		return self
 	}
 }
+
+extension NSSlider {
```

Comment:
Totally optional, but would be nice to get these `Defaults` extensions upstreamed to https://github.com/sindresorhus/Defaults after this PR is merged :)